deposit_contract.sol

```diff
@@ -101,21 +100,17 @@ contract DepositContract is IDepositContract {
         );
 
         // Compute deposit data root (`DepositData` hash tree root)
-        // These are helpers and are implicitly initialised to zero.
-        bytes16 zero_bytes16;
-        bytes24 zero_bytes24;
-        bytes32 zero_bytes32;
-        bytes32 pubkey_root = sha256(abi.encodePacked(pubkey, zero_bytes16));
+        bytes32 pubkey_root = sha256(abi.encodePacked(pubkey, bytes16(0)));
```

Comment:
We could introduce a function that does the zero-padding (depending on the length of the bytes argument) and computes sha256 - but maybe it's better like this?

Reply:
Measured it finally, the `zero_bytes16` to `bytes16(0)` change increases code size by 15 bytes, but reduces it by 58 bytes when optimized. Not sure about gas usage, because python broke locally for some reason.